clients/ui/frontend/src/app/utils/utils.ts

```diff
@@ -0,0 +1,12 @@
+export const isEnumMember = <T extends object>(
```

Comment:
We have this util present in shared library, so we can remove this as well and import it from `mod-arch-shared` wherever needed.